src/App.tsx

```diff
@@ -1,61 +1,154 @@
+import classNames from 'classnames';
+
+import { useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const todosWithUsers = todosFromServer.map(todo => {
+    const chosen = usersFromServer.find(user => user.id === todo.userId);
+
+    if (!chosen) {
+      throw new Error(`error in ${todo.id} task - no one user was found for this task`);
+    }
+
+    return {
+      ...todo,
+      user: chosen,
+    };
+  });
+
+  const [chosenUserValue, setChosenUserValue] = useState('0');
+  const [newTitle, setNewTitle] = useState('');
+  const [listWithUsers, setListWithUsers] = useState(todosWithUsers);
+  const [addClicked, setAddClicked] = useState(false);
+
+  const createTask = () => {
+    if (!newTitle.trim().length) {
+      return;
+    }
+
+    const chosenUser = usersFromServer.find(user => (
+      user.id === +chosenUserValue));
+
+    if (!chosenUser) {
+      return;
+    }
+
+    const maxId = Math.max(...listWithUsers.map(el => el.id));
+
+    setListWithUsers(list => ([
+      ...list,
+      {
+        user: chosenUser,
+        id: (maxId + 1),
+        title: newTitle,
+        completed: false,
+        userId: chosenUser.id,
+      },
+    ]));
+
+    setAddClicked(false);
+    setNewTitle('');
+    setChosenUserValue('0');
+  };
+
   return (
     <div className="App">
-      <h1>Add todo form</h1>
+      <h1 className="title">Add todo form</h1>
 
       <form action="/api/users" method="POST">
         <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
+          <label
+            htmlFor="todoTitle"
+            className="label"
+          >
+            Title
+          </label>
+          <input
+            type="text"
+            data-cy="titleInput"
+            placeholder="write the Title of todo"
+            value={newTitle}
+            id="todoTitle"
+            className={classNames('input',
+              { 'is-danger': (addClicked && !newTitle) })}
+            onChange={(e) => {
```

Comment:
Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)

Reply:
changed